backend/src/controller/seed.rs

```diff
@@ -6,41 +6,56 @@ use actix_web::{
     HttpResponse, Result,
 };
 
-use crate::{
-    config::db::Pool,
-    model::{dto::NewSeedDto, response::Body},
-    service,
-};
+use crate::{config::db::Pool, model::dto::NewSeedDto, service};
 
 /// Endpoint for fetching all [`SeedDto`](crate::model::dto::SeedDto).
 ///
 /// # Errors
 /// * If the connection to the database could not be established.
 /// * If [web::block] fails.
+#[utoipa::path(
+    context_path = "/api/seeds",
+    responses(
+        (status = 200, description = "Fetch all seeds", body = Vec<SeedDto>)
+    )
+)]
 #[get("")]
```

Comment:
I think we can omit `context_path` if we omit the route in `config::routes` and set it here in `#[get("/api/seeds")]`.
Not sure if that's the best way to do that.
I'll look into the whole PR later in the evening.

Reply:
Yes that would be possible. If you think that is easier to understand then I can change it.

Reply:
I think it would be easier to maintain since we don't have to repeat the path in two different files that way.

Reply:
Hm. Seems to be more annoying than I thought. There is no real way to specify an empty scope/empty service in actix. So I cannot group routes together, which leads to problems with the API documentation and NormalizePath.
It would technically be possible to do, but is kind of hacky, so I would prefer to leave it like it is for now.